.circleci/config.yml

```diff
@@ -31,7 +32,17 @@ workflows:
           requires:
             - publish_docker_local
             - plugins_test
+      - codacy/sbt:
+          name: publish_sonatype
+          cmd: sbt retrieveGPGKeys core/publishSigned sonatypeBundleRelease
+          # filters:
```

Comment:
restore this after trying

Reply:
Sure :)

Reply:
It worked. Reverted last commit that added the comment 👍